tests/phpunit/data/themedir1/empty-fontface-theme/theme.json

```diff
@@ -1,4 +1,5 @@
 {
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
```

Comment:
looks like extra whitespace appeared here, or a tab instead of spaces

Reply:
I can't see any different in the PR. All the lines have a single tab indent as far as I can tell.

Reply:
not sure what happened. it was different before, but maybe that was on my display or maybe Github messed up (which I'm pretty sure I've seen before).